compose.yaml

```diff
@@ -44,7 +44,7 @@ services:
       - crawl_data:/app/.scrapy
     environment:
       - OPENAI_API_KEY
-      - ENABLE_TEXTRACT
+      - ENABLE_TEXTRACT=false
```

Comment:
Can we instead remove this environment variable and just use the existence of the SYCAMORE_TEXTRACT_PREFIX to decide that we are enabling textract?

Changing the environment variable works, but it's annoying from a user experience, I have to set two variables and they're redundant. I want to make sure the user experience of using the containers is as easy as possible since that's the first place people will see our stuff.